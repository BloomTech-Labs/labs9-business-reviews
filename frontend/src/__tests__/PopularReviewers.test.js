import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import React from 'react';
import PopularReviewersComponent from '../components/PopularReviewers';

describe('PopularReviewersComponent />', () => {
  it('renders', () => {
    shallow(<PopularReviewersComponent />);
  });
  it('matches snapshot', () => {
    const wrapper = shallow(<PopularReviewersComponent />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
  it('should contain one h1 with text "Popular Reviewers"', () => {
    let app = shallow(<PopularReviewersComponent/>);
    expect(app.find("h1").length).toBe(1);
    expect(app.find("h1").text()).toMatch("Popular Reviewers");
  });
});
