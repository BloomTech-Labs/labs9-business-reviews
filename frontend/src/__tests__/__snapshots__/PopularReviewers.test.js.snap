// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PopularReviewersComponent /> matches snapshot 1`] = `
<styled.div>
  <div
    id="Reviewers"
  />
  <h1>
    Popular Reviewers
  </h1>
</styled.div>
`;
